packages/malloy-interfaces/src/to_malloy.ts

```diff
@@ -505,6 +524,24 @@ function havingToFragments(having: Malloy.FilterOperation[]): Fragment[] {
   return formatBlock('having', having.map(filterOperationItemToFragments), ',');
 }
 
+function calculateToFragments(
+  calculate: Malloy.CalculateOperation[]
+): Fragment[] {
+  const fragments: Fragment[] = [];
+  for (let i = 0; i < calculate.length; i++) {
+    const operation = calculate[i];
+    if (i !== 0) {
+      fragments.push(NEWLINE);
+    }
+    if (operation.name) {
+      fragments.push(maybeQuoteIdentifier(operation.name));
+      fragments.push(' is ');
+    }
+    fragments.push(...fieldToFragments(operation.field));
+  }
+  return formatBlock('calculate', [fragments]);
+}
```

Comment:
I would try to make `aggregateToFragments`, `groupByToFragments`, and `calculateToFragments` share code at this point, because they're all basically the same (I see some minor differences but they may be able to be ignored, not sure).